include/common.h

```diff
@@ -3,4 +3,87 @@
 
 #include "types.h"
 
+typedef struct {
+    u32* unk0;
+    u32* unk4;
+    u32* unk8;
+    u32 unkC;
+    u32 unk10;
+    u16 unk14;
+    u16 unk16;
+    u32 unk18;
+    u32 unk1C;
+    u32 unk20;
+    u32 unk24;
+    u32 unk28;
+    u32 unk2C;
+} Unk80132ED0;
+
+typedef struct {
+    Unk80132ED0* unk0;
+    f32 position[3];
+    u32* pointerToSyscallEntry;
+    u32 unk14;
+    f32 unk18[3];
+    f32 unk24;
+    f32 unk28;
+    f32 unk2C;
+    u32 unk30;
+    f32 unk34;
+    f32 scale;
+    u32 unk3C;
+    u32 unk40;
+    f32 rotation[3];
+    f32 unk50;
+    f32 unk54;
+    f32 unk58;
+    u8 unk5C;
+    u8 unk5D;
+    u8 unk5E;
+    u8 unk5F;
+    f32 unk60;
+    u32 unk64_20 : 12;
+    u32 unk64_19 : 1;
+    u32 unk64_18 : 1;
+    u32 unk64_17 : 1;
+    u32 unk64_16 : 1;
+    u32 unk68;
+    u32 unk6C_24 : 7;
+    u32 unk6C_16 : 9;
+    u32 unk6C_8 : 7;
+    u32 unk6C_0 : 9;
+    u16 unk70;
+    u16 unk72;
+    u32 unk74_31 : 1;
+    u32 unk74_30 : 1;
+    u32 unk74_16 : 14;
+    u32 unk74_7 : 9;
+    u8 unk78;
+    u8 unk79_4 : 4;
+    u8 unk79_3 : 1;
+    u8 unk79_2 : 1;
+    u8 unk79_1 : 1;
+    u8 unk79_0 : 1;
+    u16 unk7A;
+    u32 unk7C_13 : 19;
+    u32 unk7C_12 : 1;
+    u32 unk7C_14 : 12;
+    u16 unk80;
+    s16 IndexForMemory;
+    u8 unk84[0x18];
+} Actor;
```

Comment:
If Actor is being moved here, then the Actor struct in src/core2/1EDAEA0.c should be removed. Also, please add offset comments to the members in this struct like that one has, e.g. `/* 0x04 */ f32 position[3];`, as they're very useful for matching once members start getting named.